app/index.html

```diff
@@ -9,8 +9,8 @@
 
     {{content-for "head"}}
 
-    <link rel="stylesheet" href="{{rootURL}}assets/vendor.css">
-    <link rel="stylesheet" href="{{rootURL}}assets/frolfr-web.css">
+    <link integrity="" rel="stylesheet" href="{{rootURL}}assets/vendor.css">
```

Comment:
What's this?

Reply:
I believe this is needed for the ember-sri addon (SubResource Integrity). See https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity